README.md

```diff
@@ -298,9 +298,9 @@ intended to be used as a tool when developing new loggers.
 [training.logger]
 @loggers = "spacy.ChainLogger.v1"
 logger1 = {"@loggers": "spacy.PyTorchLogger.v1", "prefix": "pytorch"}
-logger2 = {"@loggers": "spacy.LookupLogger.v1", "substring": "pytorch"}
+logger2 = {"@loggers": "spacy.LookupLogger.v1", "patterns": ["^(p|P)ytorch"]}
```

Comment:
It would feel more straightforward to me to work with `[pP]` rather than use a capturing group `(p|P)` here.

Reply:
Done.